### PR TITLE
New Events page - AI Workshop

### LINKED ISSUE #1896: 📄 New Events page - AI Workshop
### 1. People in the loop:

@ae-cher-ssw @calumjs @jernejk @adamcogan 
As per email subject: RE: Approval for new training - AI 2-day Workshop

### 2. Page template to copy:

Events

### 3. CRM Record (only for consulting service pages):

N/A

### 4. URI:

ai-workshop

### 5. Page Title:

AI Tools 2-day Workshop

### 6. Meta description:

2-day AI tools training workshop for developers: Generative AI, prompt engineering, ChatGPT DALLE, GitHub Copilot, and the OpenAI API. Led by experts.

### 7. Related URLs:

N/A

### 8. Content:

Content in this doc: https://sswcom-my.sharepoint.com/:w:/g/personal/sethdaily_ssw_com_au/Ea-45LJIr15OhX46GJ1jBa0BqYSxCR8UFRmkPVxqDAsTvg?e=Qw4x6Z

### PR DESCRIPTION
**Pull request by sethdaily**
- Affected routes: /events/ai-workshop
- Fixed #1896 

- [x] If adding a new page, I have followed the [📃 New Webpage](https://github.com/SSWConsulting/SSW.Website/issues/new?assignees=&labels=&projects=&template=new_webpage.yml&title=%F0%9F%93%84+%7B%7B+TITLE+%7D%7D+) issue template

### CODE REVIEW

File: content/events/ai-workshop.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+            </>}
+            icon="/images/icons/university_hat.png"
+          />
+    _template: Agenda
+  - header: Presenters
+    presenterList:
+      - presenter: content/presenters/calum-simpson.mdx
+      - presenter: content/presenters/jernej-kavka.mdx
+    _template: PresenterBlock
+  - content: "## Introduction\n\nSSW is thoroughly exploring the evolving field of AI for software development and everyone at SSW is encouraged to use AI in their work. Since Generative AI has become widespread, developers taking advantage of it are building better software faster and SSW’s productivity has risen. This workshop aims to share the knowledge of these great modern tools to help others.\n\n\_\n\n## Who Should Attend\n\nDevelopers of all skill levels with any level of AI experience.\n\n\_\n\n## What’s Included?\n\n\\- Directed learning with hands-on exercises\n\n\\- Mentoring from SSW experts\n\n\\- Source code and project templates for your reference\n\n\\- Catering (Morning tea, lunch & afternoon tea)\n"
```
Comment (Harry-Ross) [inline content/events/ai-workshop.mdx]: Can you fix the bullet points here, they aren't displaying correctly

File: content/events/ai-workshop.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+---
+seo:
+  title: AI Tools 2-day Workshop
+  description: >-
+    2-day AI tools training workshop for developers: Generative AI, prompt
+    engineering, ChatGPT DALLE, GitHub Copilot, and the OpenAI API. Led by
+    experts.
+eventHeader:
+  heroBackground: /images/background/polygonBackground.png
+  imgOverlay: /images/background/SSWDarkMode_Logo.png
```
Comment (Harry-Ross) [inline content/events/ai-workshop.mdx]: image

Image is very big

File: content/events/ai-workshop.mdx (hunk 1 above)
Comment (sethdaily) [reply content/events/ai-workshop.mdx]: done

File: content/events/ai-workshop.mdx (hunk 2 above)
Comment (sethdaily) [reply content/events/ai-workshop.mdx]: done - replaced it with a different banner with a smaller logo
Note: image is a placeholder for the real design

**Comment by camillars** [comment]
@sethdailyssw 

1. the eventbrite links are not generated yet - please add
2. speakers - missing bio
3. missing location venues (@Harry-Ross can we make this part of the template for events pages?)
4. do we need any computer setup for this training? Check with Calum & JK
5. the dates still need to be fixed as they clash with public holidays

thanks.

**Comment by sethdaily** [comment]
@camillars 

>1. the eventbrite links are not generated yet - please add
Done

>2. speakers - missing bio
Not done - as per our conversation we are removing bios (email: RE: .NET Superpowers Retro)

>3. missing location venues (@Harry-Ross can we make this part of the template for events pages?)
Done - added

>4. do we need any computer setup for this training? Check with Calum & JK
In progress - but this shouldn't hold up publishing the page

>5. the dates still need to be fixed as they clash with public holidays
Done - new dates approved by Calum and JK

**Review by bradystroud — APPROVED** [review]
LGTM

**Pull request merged**